### ISSUE TITLE
imagens corrompidas - desafio social

### ISSUE THREAD

**Issue by rebecaborba** [gustavoguanabara/html-css#578]
Bom dia, estou fazendo o desafio mas as fotos estão corrompidas, não abrem nem no PC e nem no código

**Comment by rebecaborba** [comment]
Ei, já baixei várias vezes mas ainda consta o mesmo erro, já até baixei aplicativo pra tentar arruma-las mas não deu certo. Você tem elas no seu repositório?

**Comment by BenjamimCS** [comment]
Provável ter algo de errado na sua rede ou no driver de rede pra ta corrompendo esses arquivos. Já tive algo assim e era só com arquivos `.zip`.

Se você fez tudo corretamente provável que nem pelo meu repositório funcione. Mas tenho sim, a vontade.

**Comment by flameastro** [comment]
> @rebecaborba Para resolver esse problema, você deve baixar o [repositório completo](https://github.com/gustavoguanabara/html-css/archive/refs/heads/master.zip) e navegar até a parte que você precisa.

Obrigado! Se isso resolveu o seu problema, por favor, não esqueça de fechar essa issue!
